contracts/test/proxy/Destroyable.sol

```diff
@@ -1,40 +0,0 @@
-// SPDX-License-Identifier: AGPL-3.0-or-later
-
-pragma solidity ^0.8.0;
-
-
-import "../../contracts/proxy/Upgradeable.sol";
-
-
-/**
-* @dev Contract that could be destroyed by selfdestruct
-*/
-contract Destroyable is Upgradeable {
```

Comment:
I didn't remember this, at all

Reply:
I didn't know it existed 😅 . 

A test failed after the update to `ape`, and I checked with @vzotova, who indicated that it was never used.